cloud/google/update_etcd_vm_image.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# TODO(alcutter): Factor out common code with update_mirror_vm_image.sh script
```

Comment:
This script just recreates what's there? Might be worth a comment on how safe it is to run.